requirements/base.in

```diff
@@ -30,6 +30,9 @@ django-cors-headers
 # DRF-spectacular for api documentation
 drf-spectacular
 django-tables2
+django-filter
```

Comment:
the changes in here won't take effect until you compile requirements `inv requirements`